usaspending_api/api_contracts/contracts/v2/autocomplete/accounts/a.md

```diff
@@ -3,16 +3,16 @@ HOST: https://api.usaspending.gov
 
 # Availability Type Code [/api/v2/autocomplete/accounts/a/]
 
-This endpoint powers the USAspending.gov Availability Type Code (A) autocomplete in the Advanced Search -> Program Source -> Treasury Account filter.  It returns the list of potential Availability Type Codes narrowed by other components supplied in the Treasury Account filter.
+This endpoint powers the USAspending.gov Availability Type Code (A) autocomplete in the Advanced Search -> Program Source -> Treasury Account filter.
 
-## POST
+## List of potential Availability Type Codes [POST /api/v2/autocomplete/accounts/a/]
```

Comment:
So this is a debate for the larger team (@tony-sappe, @ebdabbs, @BrianZito, etc), but should we be putting the description in the method title or on the line below like other descriptive text?  For example:

```
## [POST]

List of potential availability type codes.

...
```

I get the desire to have a description per method since a contract can contain multiple.  Ours usually don't (unless you count OPTIONS), but on an updatable site it's very common to support multiple methods per URL.

Also, does it make sense to repeat the URL?  I was trying to move away from that since the URL is usually specified at the top of contract.

Reply:
I am fine to not repeat the URL the only issue I noticed is on a route where there are parameters in the URL. It might get confusing to have a GET /this/endpoint/{var1}/
and a GET /this/endpoint/ on the same page without specifying somehow that one is a specific entity and the other is a list.

Reply:
Have we tested Dredd using different method URLs in the same contract where we have the primary URL listed at the top?  Is it able to figure which to call correctly?

Another option would be to only include the method URL if it differs from the document level URL.

Anyhow, the duplicated URL isn't a showstopper.

Reply:
Per a comment Tony had further up and this comment I have created a separate file if we use a variable in the URI of the endpoint, but not if we use a variable as query parameters. This way we can use simple GET, POST, etc. names for Header 2 with a short description underneath.